Tests/Behaviors/SnapToPointLayoutBehaviorTests.swift

```diff
@@ -356,6 +356,54 @@ class SnapToPointLayoutBehaviorTests: XCTestCase {
 
     }
 
+    func testAttributesFilter() {
+        let behavior = SnapToPointLayoutBehavior(scrollDirection: .Horizontal(.Center))
 
+        let layout = UICollectionViewLayout()
+        behavior.getFilteredAttributes(layout: layout, frame: CGRect(x: 0, y: 0, width: 480, height: 360))
 
+
+        let collection = CollectionViewController()
```

Comment:
This test can be much simpeler by not using a `CollectionViewController`

Checkout `BrickCollectionViewTests:testWithImageInCollectionBrick`
```
        let section1 = BrickSection(bricks: [
            ImageBrick(width: .Ratio(ratio: 1/4), height: .Ratio(ratio: 1), dataSource: ImageBrickModel(image: image, contentMode: .ScaleAspectFill)),
            ])

        let section = BrickSection(backgroundColor: .whiteColor(), bricks: [
            CollectionBrick("Collection 1", backgroundColor: .orangeColor(), scrollDirection: .Horizontal, dataSource: CollectionBrickCellModel(section: section1), brickTypes: [ImageBrick.self]),
            ])
        brickView.setupSectionAndLayout(section)

        let cell1 = brickView.cellForItemAtIndexPath(NSIndexPath(forItem: 0, inSection: 1))
        XCTAssertEqual(cell1?.frame, CGRect(x: 0, y: 0, width: 320, height: 80))
```